### PR TITLE
Man page

### PR DESCRIPTION
**Pull request by mbjd**
I've made a simple man page, it pretty much has the same information as the README. Here's what it looks like:

```
DIARY(1)                    General Commands Manual                   DIARY(1)

NAME
       diary - Simple text-based diary program

SYNOPSIS
       diary [DIRECTORY]

DESCRIPTION
       diary is a simple text-based program for managing diary entries.

ENVIRONMENT
       DIARY_DIR
              If this variable is set to a directory that can be opened, diary
              will use it to store diary files. Diary files  are  simple  text
              files  named  after  their  date, formatted like YYYY-MM-DD. All
              other files are ignored.

       EDITOR The program used to edit diary entries.

ARGUMENTS
       If the argument DIRECTORY is given,  diary  files  are  read  from  and
       stored to that directory, ignoring the DIARY_DIR environment variable.

NAVIGATION
       Navigation is done using the following vim-inspired keyboard shortcuts:

       Key(s)        Action
       ======        ======
       k, up         go backward by 1 week
       j, down       go forward by 1 week
       h, left       go backward by 1 day
       l, right      go forward by 1 day
       J             go forward by 1 month
       K             go backward by 1 month

       e, enter      edit current entry
       d, x          delete current entry
       q             quit the program

       N             go to the previous diary entry
       n             go to the next diary entry
       g             go to the first date
       G             go to the last date

       t             jump to today
       s             jump to specific day

DEVELOPMENT
       All   source   code   is   available   in   this   github   repository:
       <https://github.com/in0rdr/diary/>. Contributions are always welcome!

                                                                      DIARY(1)
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,6 +33,10 @@ clean:
 
 install: $(TARGET)
 	cp $(TARGET) $(BINDIR)/$(TARGET)
+	mkdir -p $(MANDIR)/man1
+	cp -f $(MAN1) $(MANDIR)/man1
+	chmod 644 $(MANDIR)/man1/$(MAN1)
```
Comment (livibetter) [inline Makefile]: These three lines L36-38 can be done with just (untested):

```sh
install -D -m 644 $(MAN1) $(MANDIR)/man1/$(MAN1)
```

File: diary.1 [hunk 2]
```diff
@@ -0,0 +1,60 @@
+.TH DIARY 1
+.SH NAME
+diary - Simple text-based diary program
+
+.SH SYNOPSIS
+.B diary
+[\fIDIRECTORY\fR]
```
Comment (livibetter) [inline diary.1]: The brackets should be angle brackets since it's not optional (yet?).

Comment (mbjd) [inline diary.1]: It is optional - You can omit the argument and supply the directory via the environment variable `DIARY_DIR`. [This](https://github.com/in0rdr/diary/blob/0f07c514f4b762a726b8c3c3d76d899894d1164d/diary.c#L298) is where the magic happens.

File: Makefile (hunk 1 above)
Comment (mbjd) [inline Makefile]: Thanks - It works just like you'd expect. I've made another commit

File: diary.1 (hunk 2 above)
Comment (livibetter) [inline diary.1]: @mbjd I totally forgot the environment variable.

**Comment by in0rdr** [comment]
Very nice, thanks.

@shaggytwodope: [tag/v0.3](https://github.com/in0rdr/diary/releases/tag/v0.3)

**Pull request merged**